scripts/override-configuration_if_needed.sh

```diff
@@ -62,29 +49,11 @@ while [ "$1" != "" ]; do
     shift
 done
 
-##################################
-# Checout assets
-##################################
-if [ -e "${CONFIGURATION_LOCATION}" ]; then
-    pushd ${CONFIGURATION_LOCATION} &> /dev/null
-    echo "Pulling configuration..."
-    git stash --include-untracked # Stash in case there are some changes here
-    git pull
-    popd &> /dev/null
-else
-    git ls-remote "${REPO_URL}" &> /dev/null
-    if [ "$?" -ne 0 ]; then
-        echo "❌ Cannot access configuration repository!"
-        exit -1
-    fi 
-
-    echo "✅ Cloning assets from ${REPO_URL} (branch: ${BRANCH}) to path ${CONFIGURATION_LOCATION}"
-    git clone --branch ${BRANCH} --depth 1 ${REPO_URL} ${CONFIGURATION_LOCATION}
-fi
-
 if [ ! -z "${OVERRIDES_DIR}" ]; then
```

Comment:
I don't see any rm of Configuration, is there a chance we copy some files to it but don't replace the ones in wire-ios/configuration.

I want to avoid the case where one config gets mixed with a previous one

Reply:
actually it relies on the fact that we have the default configuration and then override some of the files if needed, see wire-ios-build-assets

Reply:
hehe, so that was good in a previous version?

Reply:
that is good, but i still think we should attach with submodule the private configs.
+ add Developer-Flags